typings.d.ts

```diff
@@ -53,7 +53,7 @@ interface HtmlWebpackPluginOptions {
      * HTML Minification options
      * @https://github.com/kangax/html-minifier#options-quick-reference
      */
-    minify: boolean | {},
+    minify: undefined | boolean | {},
```

Comment:
All properties are optional so we don't need to add the undefined here :)

Reply:
Without this I got a type error. I can try changing it to optional instead?

Reply:
Oh sorry you are absolutely right - usually all options are non optional after the constructor.  
But as we don't know the compiler state at this time your solution is absolutely correct.

Reply:
Would you prefer optional or a union with undefined?